### ISSUE TITLE
Queue import error

### ISSUE THREAD

**Issue by euri10** [pipermerriam/ethereum-client-utils#1]
hello,
I'm trying to make populus work and got this error, is it intended to run on python 2 ?

```
➜  testpopulus git:(master) ✗ populus deploy
Traceback (most recent call last):
  File "/usr/local/bin/populus", line 9, in <module>
    load_entry_point('populus==0.7.4', 'console_scripts', 'populus')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 568, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2720, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2380, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2386, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/populus/cli/__init__.py", line 2, in <module>
    from .attach_cmd import attach  # NOQA
  File "/usr/local/lib/python3.5/dist-packages/populus/cli/attach_cmd.py", line 16, in <module>
    from eth_rpc_client import Client
  File "/usr/local/lib/python3.5/dist-packages/eth_rpc_client/__init__.py", line 1, in <module>
    from .client import Client  # NOQA
  File "/usr/local/lib/python3.5/dist-packages/eth_rpc_client/client.py", line 3, in <module>
    from eth_client_utils import (
  File "/usr/local/lib/python3.5/dist-packages/eth_client_utils/__init__.py", line 1, in <module>
    from .client import (  # NOQA
  File "/usr/local/lib/python3.5/dist-packages/eth_client_utils/client.py", line 4, in <module>
    import Queue
ImportError: No module named 'Queue'
```

**Comment by pipermerriam** [comment]
Yes, but python3 support is becoming a high priority pretty quickly.  I believe fixing this _should_ be as simple as including `six` in the dependency chain and then running the test suite in python3 and seeing what's broken.

If you'd like to take a stab at I'd be happy to help in any way I can.

**Comment by pipermerriam** [comment]
Fixed and up on pypi as `0.3.1`

The dependent libraries are going to take a little longer to get python3 support into but I'm working on it.
